app/src/main/java/com/arygm/quickfix/ui/home/MessagesScreen.kt

```diff
@@ -272,3 +525,24 @@ fun shouldShowDateDivider(previousTimestamp: Timestamp?, currentTimestamp: Times
   val dateFormat = SimpleDateFormat("yyyyMMdd", Locale.getDefault())
   return dateFormat.format(previousDate) != dateFormat.format(currentDate)
 }
+
+@Composable
+fun SuggestionsRow(suggestions: List<String>, onSuggestionClick: (String) -> Unit) {
+  LazyRow(
+      modifier = Modifier.fillMaxWidth().padding(vertical = 0.dp),
+      horizontalArrangement = Arrangement.spacedBy(16.dp),
+      contentPadding = PaddingValues(horizontal = 8.dp)) {
+        items(suggestions) { suggestion ->
+          Button(
+              onClick = { onSuggestionClick(suggestion) },
+              colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF800020)),
+              shape = RoundedCornerShape(16.dp),
+              modifier = Modifier.padding(vertical = 8.dp)) {
```

Comment:
Same thing here, I think you can pass `maxWidth` and `maxHeight` as parameters of this composable and use widthRatio and HeightRatio to make padding responsive :)